metrics/grafana/common.py

```
@@ -0,0 +1,1828 @@
+        mode="spectrum",
+        max=None,
+        min=None,
+    )
+
+
+def heatmap_panel(
+    title: str,
+    metric: str,
+    description=None,
+    instance_selector: Sequence[str] = CPP_LABEL_SELECTORS,
+    label_selectors: list[str] = [],
```

Ruff (0.16.1)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)